### PR TITLE
update sequel version to 5.0

### PR DESCRIPTION
**Pull request by jimiray**
Bumps the Sequel version up to 5.0

### CODE REVIEW

File: lib/sinja/sequel/version.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Sinja
   module Sequel
-    VERSION = '0.1.6'
+    VERSION = '0.1.7'
```
Comment (mwpastore) [inline lib/sinja/sequel/version.rb]: I'll bump the version number when I do a release. You can take this change out. Thank you!

File: sinja-sequel.gemspec [hunk 2]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3.0'
 
-  spec.add_dependency 'sequel', '~> 4.44'
+  spec.add_dependency 'sequel', '~> 5.0.0'
```
Comment (mwpastore) [inline sinja-sequel.gemspec]: I don't want to preclude people from using Sequel 4.49 (with deprecation warnings blazing away) if they're not ready to go to Sequel 5.0 for some other reason. Can you change this to `'>= 4.49', '< 6'`? Thank you!

**Comment by jimiray** [comment]
@mwpastore all fixed up.

**Comment by mwpastore** [comment]
Looks great! Thanks for the PR.

**Pull request merged**